test/Driver/Gnu.C

```diff
@@ -0,0 +1,25 @@
+//------------------------------------------------------------------------------
+// CLING - the C++ LLVM-based InterpreterG :)
+//
+// This file is dual-licensed: you can choose to license it under the University
+// of Illinois Open Source License or the GNU Lesser General Public License. See
+// LICENSE.TXT for details.
+//------------------------------------------------------------------------------
+
+// RUN: cat %s | %cling -D__STRICT_ANSI__ -std=gnu++11 -Xclang -verify 2>&1 | FileCheck %s
+// RUN: cat %s | %cling -std=gnu99 -x c -Xclang -verify 2>&1 | FileCheck %s
```

Comment:
Could you add a test for -std=c++1z? Reason is that cling cannot be compiled with both 1z and 11, so this will check that we don't override the -std by whatever cling detects at build time. (Yes, one could argue gnu99 could play that role, but that has a language change included; I'd prefer to test these two items separately.)